.github/workflows/test.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
           POSTGRES_USER: postgres
           POSTGRES_DB: hollo_test
         options: >-
-          --health-cmd pg_isready
+          --health-cmd "pg_isready -U ${POSTGRES_USER}"
```

Comment:
This fixes all those `FATAL: role "root" does not exist` errors in the CI StopContainer logs.

https://github.com/fedify-dev/hollo/actions/runs/14947414516/job/41992559600?pr=130#step:16:60